di-kodein/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+val ktor_version: String by project
```

Comment:
Using the Gradle version catalog could help with an issue of having multiple Kotlin versions